tnth/prodanon/anonymize.py

```diff
@@ -0,0 +1,31 @@
+import importlib
+import os
+from prodanon.conn import Conn
+
+def test_connection(cursor):
+    cursor.execute("SELECT version();")
+    version = cursor.fetchone()
+    assert version[0].startswith('PostgreSQL ')
+
+
+package = "prodanon"
+transformers_dir = os.path.join(os.path.dirname(__file__), package, "transformers")
+custom_dir = os.path.join(os.path.dirname(__file__), package, "custom")
+
+if __name__ == '__main__':
+    print("Begin Anonymize Data")
```

Comment:
If I understand correctly, this utility will delete/pave over any existing DB.

eproms-stage seems like a good candidate for an existing deploy, but would you suggest creating a new deploy for this process?

Reply:
I believe we need a flow that:
1. exports eproms-prod db (pgdump...)
2. imports that dump file into this project's docker volume
3. runs the utility
4. exports that very modified db
5. imports the result of the previous step on eproms-test

I was hoping you might suggest the best path ;)  I'm very paranoid about destroying prod on prod, thus the desire to copy it out of the way, first.